lib/vm-run

```diff
@@ -319,7 +319,7 @@ vm::uefi(){
         _bootrom="${_bootrom},${VM_DS_PATH}/${_name}/uefi-vars.fd"
     fi
 
-    _opts="-Hwl bootrom,${_bootrom}"
+    _opts="${_opts} -l bootrom,${_bootrom}"
```

Comment:
While bhyve's manpage states that -w is intended for debugging purposes only, it's required by some guests. Especially, Windows VM's don't boot on most systems properly without -w.

Reply:
OK, let's move it to the default options then. It's not specific to UEFI.